src/CloudFlareConfigInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudflareConfigInterface.
+ */
+
+namespace Drupal\cloudflare;
+
+/**
+ * Zone methods for CloudFlare.
+ */
+interface CloudFlareConfigInterface {
```

Comment:
Overall, it feels *extremely* weird to have this kind of "pseudo config" object, since Drupal 8 has an explicit configuration system already. When you then also look at the method names, it's quite clear this is not at all config-related.

This suggests to me that some of the abstractions may be wrong?